sporkfish/searcher/minimax.py

```diff
@@ -50,14 +66,27 @@ def evaluator(self) -> Evaluator:
         return self._evaluator
 
     def _ordered_moves(self, board: Board, legal_moves: Any) -> Any:
-        # order moves from best to worse
+        """
+        Order the given legal moves from best to worst based on a move ordering heuristic.
+
+        This method sorts the legal moves based on a move ordering heuristic, which aims to prioritize
+        moves that are likely to lead to better positions or outcomes. The ordering can help improve
+        the efficiency of the search algorithm by considering more promising moves first.
+
+        :param board: The current state of the chess board.
+        :type board: chess.Board
+        :param legal_moves: The legal moves to be ordered.
+        :type legal_moves: Any
+        :return: The ordered legal moves.
+        :rtype: Any
+        """
         return sorted(
             legal_moves,
             key=lambda move: (self._move_order.evaluate(board, move),),
             reverse=True,
         )
 
-    def _aspiration_windows_search(
+    def _aspiration_windows(
```

Comment:
I prefer the previous name actually. Anything that returns a move, in my head reads like a search to me.

LMKYT

Reply:
Ok

Reply:
Done!